Makefile

```diff
@@ -273,8 +272,9 @@ generate-swagger: $(SWAGGER_SPEC) ## generate the swagger spec from code comment
 
 $(SWAGGER_SPEC): $(GO_SOURCES_NO_BINDATA)
 	$(GO) run $(SWAGGER_PACKAGE) generate spec -x "$(SWAGGER_EXCLUDE)" -o './$(SWAGGER_SPEC)'
-	$(SED_INPLACE) '$(SWAGGER_SPEC_S_TMPL)' './$(SWAGGER_SPEC)'
-	$(SED_INPLACE) $(SWAGGER_NEWLINE_COMMAND) './$(SWAGGER_SPEC)'
+	$(SED_INPLACE) -e '$(SWAGGER_SPEC_S_TMPL)' './$(SWAGGER_SPEC)'
+	$(SED_INPLACE) -e 's/\r$$//' './$(SWAGGER_SPEC)' # convert crlf to lf
+	$(SED_INPLACE) -e '$$a\' './$(SWAGGER_SPEC)' # add final newline
```

Comment:
```suggestion
```

Reply:
Yes, no final new line is the expected result. Remove the non-sense final new line forever.

Reply:
> This worked at my first try, but no idk what this implies (crlf to lf not needed on my end)

But I do not think the final new line makes sense here. The file content is just a generated JSON output, no one would really edit it.